### PR TITLE
fix: driver(country,state), vehicle(vendor,owner operator)

### PR DESCRIPTION
**Pull request by Hardeep8587**
## Change Description

1. driver(country,state) library in modal of driver
2.  vehicle(vendor,owner operator) - updated detail page and add form

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/fleet/drivers/add-driver/add-driver.component.ts [hunk 1]
```diff
@@ -868,46 +875,46 @@ fetchCities(address){
...
-              geoCords: {
-                lat: result.address[i].geoCords.lat,
-                lng: result.address[i].geoCords.lng
-              },
-              userLocation: result.address[i].userLocation
-            });
-          }
-        }
-        this.driverData.address = this.newAddress;
+        // for (let i = 0; i < result.address.length; i++) {
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/drivers/add-driver/add-driver.component.ts]: Is this code required ? If not please remove.

File: src/app/pages/fleet/vehicles/vehicle-detail/vehicle-detail.component.ts [hunk 2]
```diff
@@ -357,16 +361,16 @@ export class VehicleDetailComponent implements OnInit {
...
         result = result.Items[0];
 
-        if(result.ownerOperatorID != '' && result.ownerOperatorID != undefined) {
-          this.apiService.getData(`ownerOperators/${result.ownerOperatorID}`).subscribe((result2: any) => {
-            let data = result2.Items[0];
-            this.ownerOperatorName = data.firstName+' '+data.lastName;
-          });
-        }
-
+        // if(result.ownerOperatorID != '' && result.ownerOperatorID != undefined) {
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/vehicles/vehicle-detail/vehicle-detail.component.ts]: Remove this code

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
Please address review comments

File: src/app/pages/fleet/vehicles/vehicle-detail/vehicle-detail.component.ts (hunk 2 above)
Comment (Hardeep8587) [reply src/app/pages/fleet/vehicles/vehicle-detail/vehicle-detail.component.ts]: I tested at my end, I kept if anything happens when Rashmi tests so I would know

File: src/app/pages/fleet/drivers/add-driver/add-driver.component.ts (hunk 1 above)
Comment (Hardeep8587) [reply src/app/pages/fleet/drivers/add-driver/add-driver.component.ts]: I tested at my end, I kept if anything happens when Rashmi tests so I would know

**Review by kunalfleethawks — APPROVED** [review]

**Pull request merged**